{
  "title": "Assertion methods should not be used within the try block of a try-catch catching an Error",
  "type": "BUG",
  "status": "ready",
  "remediation": {
    "func": "Constant/Issue",
    "constantCost": "5min"
  },
  "tags": [
    "junit",
    "tests"
  ],
  "extra": {
    "replacementRules": [],
    "legacyKeys": []
  },
  "defaultSeverity": "Critical",
  "ruleSpecification": "RSPEC-5779",
  "sqKey": "S5779",
  "scope": "Tests",
  "defaultQualityProfiles": [
    "Sonar way"
  ],
  "quickfix": "unknown",
  "all_languages": [
    "java",
    "php"
  ],
  "allKeys": [
    "S5779"
  ],
  "branch": "master"
}